Backend/prisma/schema.prisma

```diff
@@ -0,0 +1,69 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+// Looking for ways to speed up your queries, or scale easily with your serverless or edge functions?
+// Try Prisma Accelerate: https://pris.ly/cli/accelerate-init
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model User {
+  id       Int      @id @default(autoincrement())
+  email    String   @unique
+  phoneNumber Int
+  password String
+  name     String?
+  bookings Booking[]
+  profilePhoto String
```

Comment:
Not requesting immediate changes on this but wanted to share some food for thought on the database design here - we can consider pulling out fields common to the User and ServiceProvider tables into separate narrower purposed tables. For instance, we could have a separate table to store just profile photos (id, profilePhoto) with the id field referencing either the User ID or the ServiceProvider ID although these ids would need to be distinct. We could follow a similar design for contact details and bookings. This will help keep tables leaner, more performant and subject specific. Let's chat more about database design principles when we sync up.

Reply:
Okay!